rijksbot

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env ruby
 
-$LOAD_PATH.unshift(File.expand_path("lib"))
+$LOAD_PATH.unshift(File.expand_path('lib'))
 require 'dotenv'
 require 'pry'
 require 'rijksbot'
 Dotenv.load
 
 client = Twitter::REST::Client.new do |config|
-  config.consumer_key        = ENV["RIJKSBOT_CONSUMER_KEY"]
-  config.consumer_secret     = ENV["RIJKSBOT_CONSUMER_SECRET"]
-  config.access_token        = ENV["RIJKSBOT_ACCESS_TOKEN"]
-  config.access_token_secret = ENV["RIJKSBOT_ACCESS_SECRET"]
+  config.consumer_key        = ENV['RIJKSBOT_CONSUMER_KEY']
+  config.consumer_secret     = ENV['RIJKSBOT_CONSUMER_SECRET']
+  config.access_token        = ENV['RIJKSBOT_ACCESS_TOKEN']
+  config.access_token_secret = ENV['RIJKSBOT_ACCESS_SECRET']
 end
 
 # timeline = client.user_timeline(count: 800).map{|t| t.in_reply_to_status_id}
 bot = Rijksbot.new(client)
 
-last_checked = Time.now.utc - 86400
+last_checked = Time.now.utc - 86_400
```

Comment:
Tell me about this change!

Reply:
[Ho rubocop rubocop: "rubocop -a" :)](https://github.com/rubocop-hq/ruby-style-guide#underscores-in-numerics)